src/coreclr/utilcode/loaderheap.cpp

```diff
@@ -1304,14 +1304,17 @@ BOOL UnlockedLoaderHeap::GetMoreCommittedPages(size_t dwMinSize)
         void *pData = ExecutableAllocator::Instance()->Commit(m_pPtrToEndOfCommittedRegion, dwSizeToCommitPart, IsExecutable());
         if (pData == NULL)
         {
-            _ASSERTE(!"Unable to commit a loaderheap page");
```

Comment:
I have removed this assert since I've noticed the same CI test that uncovered the issue has failed due to this assert in another run. We should just return FALSE and let the caller handle the memory allocation failure.

Reply:
I've seen this assert occasionally on macOS-arm64 Fuzzlyn runs but have not had time to create a repro case. When would we expect this to occur? In low memory situations or for other reasons too?

Reply:
In low memory situation only.